packages/social-auth/src/forIDP.ts

```diff
@@ -3,22 +3,22 @@ import { FacebookAuth } from "./FacebookAuth";
 import { GoogleAuth } from "./GoogleAuth";
 import { RedditAuth } from "./RedditAuth";
 import { TwitterAuth } from "./TwitterAuth";
-import type { IDP } from "./types";
+import type { IDP, RedirectSurface } from "./types";
 
 /**
- * Returns the Auth instance for the given IDP
+ * Returns the Auth instance for the given IDP and redirect surface.
```

Comment:
please add a bit on what the redirect surface mean here

Reply:
Done